guide/14-deep-learning/model_extension_guide.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>But, what if you came  --&gt; remove comma after but</p><p>github --&gt; GitHub</p><p>general purpose --&gt; general-purpose</p><p>Pytorch --&gt; PyTorch</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/987/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>